helpers/db.py

```diff
@@ -0,0 +1,57 @@
+import lancedb
```

Comment:
this file is not db.py. cause it responsible for adding data. 
lets keep everything in a single file